src/plugins/saved_objects/public/save_modal/saved_object_save_modal.tsx

```diff
@@ -111,6 +112,11 @@ export class SavedObjectSaveModal extends React.Component<Props, SaveModalState>
               defaultMessage="Save {objectType}"
               values={{ objectType: this.props.objectType }}
             />
+            {this.props.objectType === 'Visualization (VisBuilder)' ? (
```

Comment:
This is the wrong condition to check - `experimental` is an actual property that should be checked, as there are other experimental objects (such as those created by the `input_control_vis` plugin).

Reply:
Sync with Josh offline and we agreed on adding an optional prop such as isExperimental for the saved object modal component. Declare a plugin level variable such as VIS_BUILDER_STATE = 'experimental' and set the isExperimental prop by checking VIS_BUILDER_STATE variable when we render the saved object modal for vis builder plugin.